.github/workflows/phpstan.yml

```diff
@@ -60,13 +53,13 @@ jobs:
 
     # https://docs.github.com/en/free-pro-team@latest/actions/guides/about-service-containers
     services:
-      mysql:
+      database:
         image: ${{ matrix.db-image }}
         env:
           MYSQL_ROOT_PASSWORD: root
         ports:
           - 3306:3306
-        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+        options: ${{ startsWith(matrix.db-image, 'mariadb') && '--health-cmd="healthcheck.sh --connect --innodb_initialized" --health-interval=1s --health-timeout=10s --health-retries=60' || '--health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3' }}
```

Comment:
can we move the command into the matrix, where we define the db image?

Reply:
Yes ❗

Reply:
Done 🍏

![image](https://github.com/staabm/phpstan-dba/assets/952007/8cb9d6f3-4f46-4008-a5f2-f4d1128ef11f)